src/cloud/digitalocean/provisioner.ts

```diff
@@ -104,8 +105,13 @@ class Provisioner {
    * @param {String} name of VM to create
    * @return {Promise.<Object>}
    */
-  public stop = (name: string) : Promise<Object> => {
-    return this.doRequest_('GET', 'droplets').then((resp: any) => {
+  public stop = (name: string): Promise<Object> => {
+    this.sendStatus_('STOP');
+    return this.doOAuth_().then((oauthObj: any) => {
```

Comment:
I believe `Object` is preferable to `any`, unless there's a special reason to return `any` (and below).